src/MLAdapters/MapMouseEvent.ts

```diff
@@ -7,7 +7,12 @@ import type { Map as MapMLGL } from "maplibre-gl";
 import type { Map as SDKMap } from "../Map";
 
 export class MapMouseEvent extends maplibregl.MapMouseEvent {
-  constructor(type: string, map: SDKMap | MapMLGL, originalEvent: MouseEvent, data: any = {}) {
+  constructor(
+    type: string,
+    map: SDKMap | MapMLGL,
+    originalEvent: MouseEvent,
+    data: any = {},
```

Comment:
would a `Record` help this error?

Reply:
https://github.com/maptiler/maptiler-sdk-js/pull/164/files#diff-e91cad2642df0ee4bd17587789b3a6fc750d7e8e1bb4f3d0139f0b60ee6ab67aR14
Yeah, makes sense. Do you like it? :D